Fixes #281 Exception thrown during installation of 1.6

cnovak opened the pull request:
Recreated menu items that conflict with new forum menu item.

cnovak committed:
Fixes #281 Exception thrown during installation: Integrity constraint violation

jacine reviewed (approved):
Tested this, and it fixes the problem! Thanks @cnovak 👍 

Only one nitpick...  Can you run `drush dcem apigee_kickstart_content` and commit all the changes? I just did this myself to test, and all the changes are safe to commit.

Each time the install runs the `uuid` changes, and that's reflected in this export, but not all the rest. It doesn't cause any errors or real issues as the uuids are not actually used for users in this case, but it's nicer for it to be consistent across all the content.

Approving this in the meantime. 😄

cnovak committed:
code review

Commit all changes from  drush default-content:export-module apigee_kickstart_content

shadcn reviewed (approved):
Tested. Installation working, uninstalling the forum module works too. LGTM
